uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -201,6 +202,11 @@ define([
       vm.canSubmit = function () {
         var canSubmit = canCalculateChange();
 
+        //check if user has changed any attribute
+        if (vm.mode == 'edit') {
+          canSubmit = canSubmit &&  !_.isEqual(initialLeaveRequest, vm.leaveRequest.attributes());
```

Comment:
Is this a requirement in terms of UX? Why not allowing the user to re-save the leave request as is, if he wants to?